### ISSUE TITLE
Consider hyper-convenient wrapper to build a single target from the console?

### ISSUE THREAD

**Issue by kendonB** [ropensci/drake#239]
I know I've asked for something similar before, and the current solution is:

```
my_config <- read_drake_config()
my_meta <- read_drake_meta()
loadd(my_target, deps = TRUE)
drake_build(target = "my_target", config = my_config, meta = my_meta)
```
This seems like it could get wrapped up into a single user-friendly function:

```
just_build_it_really_easily_with_one_line <- function(target, config = read_drake_config(), meta = read_drake_meta()){
  loadd(my_target, deps = TRUE)
  drake_build(target = target, config, meta)
}
```

**Comment by wlandau** [comment]
If you're running into this a lot, I suppose we should have an easy one-liner, maybe the following + documentation.

```r
make_single <- function(target){
  my_config <- read_drake_config()
  loadd(my_target, deps = TRUE)
  drake_build(target = "my_target", config = my_config) # `drake_build()` already gets the latest metadata from `config`
}
```

With the stream of incoming issues, I am relying more and more on pull requests from others, especially for small issues like this one.. If you have the time, I would really appreciate it.

**Comment by kendonB** [comment]
I'm not sure about the function name. Does ``drake_build`` as it is need to be exported? Can this wrapper be ``drake_build`` and what was ``drake_build`` have a name change?

**Comment by wlandau** [comment]
Good point. The internals are in flux right now, so I can try and free up the name.

**Comment by wlandau** [comment]
Fixed via #242. It was harder than I expected. I was playing around with `drake_build()`, and it kept reverting things I was doing in the console. I had to add a `replace` argument to make sure things in the user's environment are not replaced by `loadd(deps = TRUE)`.
